src/Aquifer.Jobs/Program.cs

```diff
@@ -15,6 +15,14 @@
 
 var host = new HostBuilder()
     .ConfigureFunctionsWorkerDefaults()
+    .ConfigureAppConfiguration((context, builder) => builder
+        .SetBasePath(context.HostingEnvironment.ContentRootPath)
+        .AddJsonFile("appsettings.json", optional: false, reloadOnChange: true)
+        .AddJsonFile(
+            $"appsettings.{context.HostingEnvironment.EnvironmentName}.json", 
+            optional: true, 
+            reloadOnChange: false)
```

Comment:
We'll either want to call  `.AddEnvironmentVariables()` at the end of this chain in order to get environment variable overrides _or_ move the `ConfigureFunctionsWorkerDefaults()` method to the end of the chain which might result in that being solved correctly.

To test you can create environment variables locally that match QA, debug a function that pulls in the `ConfigurationOptions` via DI, and check the values.

The generally respected order should be:
1. appsettings.json
2. {environment}.appsettings.json
3. Environment variables

Ideally we use the configure defaults method at the end because that also wires up other overrides that we don't currently use but which would keep things consistent with the other API apps.

Reply:
During testing and refactoring, I found that the respected values will put `appsettings.{env}.json` higher than `appsettings.json`. That is to say, the same kv pair will use the `appsettings.{env}.json` value if both are set.